modules/@angular/compiler/src/template_parser.ts

```diff
@@ -718,27 +720,35 @@ class TemplateParseVisitor implements HtmlAstVisitor {
     } else {
       if (parts[0] == ATTRIBUTE_PREFIX) {
         boundPropertyName = parts[1];
+        // NB: For security purposes, use the mapped property name, not the attribute name.
+        securityContext = this._schemaRegistry.securityContext(
```

Comment:
Might be better to only fill the `SecurityContext.STYLE` here, and calculate the other kinds in `_createElementPropertyAst` as we do `getMappedPropName` there already (i.e. there we calculate which property should be used for real).